### PR TITLE
chore(docstrings): updating v2.1-v3.0 conversion script docstrings to match the new task label

### PR DESCRIPTION
**Pull request by CarolinePascal**
## Type / Scope

- **Type**: Chore
- **Scope**: `convert_dataset_v21_to_v30.py`

## Summary / Motivation

Docstrings update

## Related issues

- Fixes / Closes: #2467
- Related: # (if any)

## What changed

Docstrings

## How was this tested (or how to run locally)

N/A

## Checklist (required before merge)

- [x] Linting/formatting run (`pre-commit run -a`)
- [x] All tests pass locally (`pytest`)
- [x] Documentation updated
- [x] CI is green

## Reviewer notes

- Anything the reviewer should focus on (performance, edge-cases, specific files) or general notes.
- Anyone in the community is free to review the PR.

### CODE REVIEW

**Comment by CarolinePascal** [comment]
I actually found a lightweight solution to rename the DataFrame index in the codebase so that :
- Newly created datasets will have tasks labelled with "tasks";
- Legacy datasets can still be loaded and used without changes.

I will update the PR asap !

**Comment by CarolinePascal** [comment]
@s1lent4gnt - I've updated the PR according to my previous comment + I've fixed the docstrings to actually match dataset v3.0 architecture.

File: src/lerobot/datasets/v30/convert_dataset_v21_to_v30.py [hunk 1]
```diff
@@ -173,7 +174,7 @@ def convert_tasks(root, new_root):
     tasks, _ = legacy_load_tasks(root)
     task_indices = tasks.keys()
     task_strings = tasks.values()
-    df_tasks = pd.DataFrame({"task_index": task_indices}, index=task_strings)
+    df_tasks = pd.DataFrame({"task_index": task_indices}, index=pd.Index(task_strings, name="task"))
```
[the remark this answers is not in the document]
Comment (CarolinePascal) [reply src/lerobot/datasets/v30/convert_dataset_v21_to_v30.py:177]: Well, turns out I've changed my mind.
